### DISCUSSION TITLE
PowerShell winget WIN 10 Pro

### DISCUSSION THREAD

**Discussion by Tiger348** [microsoft/winget-cli#2389] [PowerShell]
A kind member suggested I step through the trouble shooting page. I was unable to resolve my issue.

In sort when using cmd prompt , When I type winget without the fle extention ".exe" . It functions as it should. But when using PowerShell I have to type winget.exe in order to function as normal.

If I type winget in isolation it prompt windows via popup "How do yu want to open this file" image attached.

This may should trivial but I'd like to resolve if possible.

Any guidence is much appreciated.

Tiger348

![Winget  exe Association](https://user-images.githubusercontent.com/109897924/181487660-fb42d25e-2da5-47b1-b9d5-141b923dfcde.png)
![winget issue 002](https://user-images.githubusercontent.com/109897924/181487671-d052f665-69df-4202-ae3d-f5c8860dd4e7.png)
![winget Issue](https://user-images.githubusercontent.com/109897924/181487675-4c3e0229-f067-43ec-9ffd-0383641ad6df.png)

**Comment by denelon** [comment]
This looks like it could an issue with a corrupted path environment variable:
* https://github.com/microsoft/winget-cli/issues/2391
